package.json

```diff
@@ -93,6 +94,7 @@
     "husky": "^4.3.5",
     "i18next": "^15.0.8",
     "istanbul-lib-coverage": "^2.0.5",
+    "js-yaml-loader": "^1.2.2",
```

Comment:
This is already specified in `dependencies` and can safely be removed from here.